src/structures/ThreadChannel.js

```diff
@@ -253,6 +253,17 @@ class ThreadChannel extends Channel {
     return members.get(this.ownerId) ?? null;
   }
 
+  /**
+   * Fetches the message that started this thread, if any.
+   * <info>This only works when the thread started from a message in the parent channel, otherwise the promise will
+   * reject. If you just need the id of that message, use {@link ThreadChannel#id this channel's id} instead.</info>
```

Comment:
I don't see the point of that documentation for ThreadChannel#id link - especially if threads ever stop sharing IDs with their message

```suggestion
   * reject. If you just need the id of that message, use {@link ThreadChannel#id} instead.</info>
```

Reply:
If they intentionally made it like this, I doubt this will ever change, but I'll apply that suggestion